web/src/app/maintenance/entities/maintenance-properties.ts

```diff
@@ -1,9 +1,8 @@
 import {AutoupdateSchedule} from "./autoupdate-schedule";
-import {KioskVersion} from "./kiosk-version";
 import {VenueMaintenanceInfo} from "./venue-maintenance-info";
 
 export interface MaintenanceProperties {
     venues: Array<VenueMaintenanceInfo>;
     autoupdateSchedules: Array<AutoupdateSchedule>;
-    kioskVersions: Array<KioskVersion>;
-}
\ No newline at end of file
+    kioskVersions: any;
```

Comment:
We could use it { [screenId: string]: KioskVersionDetails } instead of any, to know what we can expect